libs/efivar/patches/007-use-host-makeguids.patch

```diff
@@ -0,0 +1,19 @@
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -31,6 +31,7 @@ GENERATED_SOURCES = include/efivar/efiva
+ MAKEGUIDS_SOURCES = makeguids.c util.c
+ MAKEGUIDS_OBJECTS = $(patsubst %.S,%.o,$(patsubst %.c,%.o,$(MAKEGUIDS_SOURCES)))
+ MAKEGUIDS_OUTPUT = $(GENERATED_SOURCES) guids.lds
++MAKEGUIDS_BIN ?= ./makeguids
```

Comment:
replaced by `HOSTCC="$(HOSTCC)" \`. can be removed.

Reply:
patch removed